rails/client/plugins/gather-dependencies.js

```diff
@@ -41,7 +41,7 @@ GatherDependencies.prototype.apply = () => {
     // Example: Textarea from 'terra-form' and Textarea from terra-form-textarea.
     let alias = name;
     if (aliasSet.has(alias)) {
-      alias = toTitleCase(library).replace(alias, alias);
+      alias = `${toTitleCase(library).replace(alias, '')}${alias}`;
```

Comment:
This prevents double naming. Example: While concatenating the library an component name a component such a `Textarea` would end up as `TerraFormTextareaTextarea`. This prevents the double `Textarea`

Reply:
I vote for the double naming! 😆